app/services/publishers/vacancy_filter.rb

```diff
@@ -1,7 +1,8 @@
 class Publishers::VacancyFilter
   attr_reader :managed_school_ids, :managed_organisations
 
-  def initialize(publisher, school_group)
+  def initialize(publisher, organisation)
+    school_group = (organisation if organisation.is_a?(SchoolGroup))
```

Comment:
I'm assuming this is is one of the things you're going to refactor next - but if not let's refactor it!

Reply:
This is going to change a lot in the next piece of work, will refactor it later